docs/getting-started/install-configure.md

```diff
@@ -0,0 +1,183 @@
+  * For Helm 2, make sure Tiller is initialized with sufficient permissions to
+    work on `crossplane-system` namespace.
+</div>
+<div class="tab-pane fade" id="setup-windows" markdown="1">
+* [Kubernetes cluster]
+  * [Kind]
+  * [Minikube], minimum version `v0.28+`
+  * etc.
+
+* [Helm], minimum version `v2.12.0+`.
+  * For Helm 2, make sure Tiller is initialized with sufficient permissions to
+    work on `crossplane-system` namespace.
+</div>
+</div>
+
+## Install Crossplane
+Install the alpha version of Crossplane, which is suitable for community use and testing:
+<ul class="nav nav-tabs">
+<li class="active"><a href="#install-tab-helm3" data-toggle="tab">Helm 3</a></li>
+<li><a href="#install-tab-helm2" data-toggle="tab">Helm 2</a></li>
+</ul>
+<br>
+<div class="tab-content">
+<div class="tab-pane fade in active" id="install-tab-helm3" markdown="1">
+```
+kubectl create namespace crossplane-system
+helm repo add crossplane-alpha https://charts.crossplane.io/alpha
+helm install crossplane --namespace crossplane-system crossplane-alpha/crossplane
+```
+</div>
+<div class="tab-pane fade" id="install-tab-helm2" markdown="1">
+```
+helm repo add crossplane-alpha https://charts.crossplane.io/alpha
+helm install --name crossplane --namespace crossplane-system crossplane-alpha/crossplane
+```
+</div>
+</div>
+
+## Install Crossplane CLI
+The Crossplane CLI adds a set of `kubectl crossplane` commands to simplify common tasks:
+```
+curl -sL https://raw.githubusercontent.com/crossplane/crossplane-cli/master/bootstrap.sh | bash
+```
+
+## Select Provider
+In order for Crossplane to be able to manage resources in a specific cloud
+provider, you will need to install a provider and configure it for Crossplane to use.
+<ul class="nav nav-tabs">
+<li class="active"><a href="#provider-tab-gcp" data-toggle="tab">GCP</a></li>
+<li><a href="#provider-tab-aws" data-toggle="tab">AWS</a></li>
+<li><a href="#provider-tab-azure" data-toggle="tab">Azure</a></li>
+<li><a href="#provider-tab-alibaba" data-toggle="tab">Alibaba</a></li>
+</ul>
+<br>
+<div class="tab-content">
+<div class="tab-pane fade in active" id="provider-tab-gcp" markdown="1">
+### Install GCP Provider
+```
+PACKAGE=crossplane/provider-gcp:v0.9.0
+NAME=provider-gcp
+kubectl crossplane package install --cluster --namespace crossplane-system ${PACKAGE} ${NAME}
+```
+
+### Create gcp-account-creds.json
+The following `curl | bash` script that creates a service account and grants
+access to infrastructure services. Alternatively you use the [Configure] docs
+to manually generate account credentials.
+```
+# replace this with your own gcp project id
+GCP_PROJECT_ID=TODO_REPLACE
+curl -s https://raw.githubusercontent.com/crossplane/stack-gcp-sample/master/hack/gcp-credentials.sh | PROJECT_ID=${GCP_PROJECT_ID} bash -
+```
+
+Save the resulting secret JSON to `gcp-account-creds.json` and be sure to delete it later!
+
+
+### Create a Provider Secret
+```
+PROVIDER_SECRET_NAMESPACE=crossplane-system
```

Comment:
Can we just use crossplane-system inline here and in the example below? I'm sure folks with Kubernetes experience will be able to spot that they could change it.

Reply:
fixed